plugin-3.0/pom.xml

```diff
@@ -0,0 +1,96 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+  <groupId>com.helger.maven</groupId>
+	<artifactId>maven-jaxb30-plugin</artifactId>
+	<packaging>maven-plugin</packaging>
+	<name>Maven JAXB 3.0.x Plugin</name>
+	<parent>
+		<groupId>com.helger.maven</groupId>
+		<artifactId>maven-jaxb2-plugin-project</artifactId>
+		<version>0.14.4-SNAPSHOT</version>
+	</parent>
+	<prerequisites>
+		<maven>${maven.version}</maven>
+	</prerequisites>
+	<dependencies>
+		<dependency>
+			<groupId>com.helger.maven</groupId>
+			<artifactId>maven-jaxb2-plugin-core</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>jakarta.xml.bind</groupId>
+			<artifactId>jakarta.xml.bind-api</artifactId>
+			<version>${jaxb30.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.sun.xml.bind</groupId>
+			<artifactId>jaxb-impl</artifactId>
+			<version>${jaxb30.version}</version>
+			<scope>runtime</scope>
+		</dependency>
+		<dependency>
+			<groupId>com.sun.xml.bind</groupId>
+			<artifactId>jaxb-xjc</artifactId>
+			<version>${jaxb30.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>jakarta.activation</groupId>
+			<artifactId>jakarta.activation-api</artifactId>
+			<version>2.0.0</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.maven.plugin-tools</groupId>
+			<artifactId>maven-plugin-annotations</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.maven.plugin-testing</groupId>
+			<artifactId>maven-plugin-testing-harness</artifactId>
+			<scope>test</scope>
+		</dependency>
+	</dependencies>
+	<build>
+		<resources>
+			<resource>
+				<directory>src/main/resources</directory>
+				<filtering>true</filtering>
+			</resource>
+			<resource>
+				<directory>${project.build.directory}/generated-resources/jaxb-xjc-runtime</directory>
+				<filtering>false</filtering>
+			</resource>
+		</resources>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-plugin-plugin</artifactId>
+			</plugin>
+			<plugin>
+				 <groupId>org.apache.maven.plugins</groupId>
+				 <artifactId>maven-dependency-plugin</artifactId>
+				 <executions>
+					 <execution>
+						 <id>unpack</id>
+						 <phase>generate-resources</phase>
+						 <goals>
+							 <goal>unpack</goal>
+						 </goals>
+						 <configuration>
+							 <artifactItems>
+								 <artifactItem>
+									<groupId>org.glassfish.jaxb</groupId>
+									<artifactId>jaxb-xjc</artifactId>
+									<version>${jaxb23.version}</version>
```

Comment:
jaxb23?

Reply:
I actually have no idea why the `unpack` done by this plugin is needed at all. I tried removing it and all the tests still pass, so I have removed it permanently now.